playbooks/platform-uninstall.yml

```diff
@@ -0,0 +1,123 @@
+---
+#### This playbook will uninstall ACS
+
+ - hosts: database
+   tasks:
+     - name: Stopping the database
+       ansible.builtin.service:
+         name: postgresql{% if ansible_os_family != 'Debian' %}{{ dependencies_version.postgres_major_version }}{% endif %}
+         state: stopped
+         enabled: no
+
+     - name: Removing the Packages   
+       ansible.builtin.package:
+         name: postgresql
+         state: absent
+ 
+ - hosts: activemq
+   tasks:
+     - name: Stopping the activemq
+       ansible.builtin.service:
+         name: activemq.service 
+         state: stopped
+         enabled: no
```

Comment:
Do not forget to remove the systemd unit file as well. This one is not installed by a package so needs to be removed  by the playbook (and systemd reloaded using an handler)